templates

```diff
@@ -969,6 +1002,15 @@
         { key => "+HDD_1", value => "disk_%FLAVOR%_%MACHINE%.qcow2" },
       ],
     },
+    {
+      name => "check_flatpak_extra_languages",
```

Comment:
Please insert this chunk in alphabetical order in this array in the `templates` file.